CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## Unreleased
+ - Add MSSQL (Microsoft SQL Server) instrumentation (supports [mssql](https://www.npmjs.com/package/mssql), version >= 4 via [tedious driver](https://www.npmjs.com/package/tedious)).
```

Comment:
Is `tedious` something that is configurable or why is this explicitly mentioned?

Reply:
Yes, there is one alternative driver [msnodesqlv8](https://www.npmjs.com/package/msnodesqlv8) (which only works on Windows, though). Since we instrument the mssql package and not the driver, in theory it should also work with msnodesqlv8 but I haven't tested it at all, that's why the tedious driver is mentioned explicitly. The motivation for this PR is an actual customer requirement with the combination mssql/tedious.